brightnessctl.c

```diff
@@ -351,7 +362,38 @@ void apply_value(struct device *d, struct value *val) {
 	d->curr_brightness = new;
 }
 
-bool write_device(struct device *d) {
+#ifdef ENABLE_SYSTEMD
+
+bool sd_write_device(struct device *d) {
```

Comment:
Is `sd_` not typically reserved for systemd's own namespace? Seems awfully prone to clashes. Perhaps `logind_set_brightness` is better?

Reply:
Very good point, thanks